web/app/webpack.config.js

```diff
@@ -64,10 +72,17 @@ module.exports = smp.wrap({
   },
   plugins: [
     // new BundleAnalyzerPlugin(), // uncomment to analyze bundle size
+    new CleanWebpackPlugin(['dist']),
+    new HtmlWebpackPlugin({
+      inject: false,
+      filename: "index_bundle.js.out",
+      template: 'lodash/index_bundle.js.lodash.tmpl',
+    }),
     new LodashModuleReplacementPlugin({
       // 'chain': true,
       'collections': true,
       'paths': true
-    })
+    }),
+    new WebpackMv()
```

Comment:
tioli we can name this WebpackMvPlugin to match what seems to be the style of the other plugin names